rezip.py

```diff
@@ -31,7 +36,13 @@
     # Read and re-zip the file in memory
     with ZipFile(source, 'r') as source_zip, ZipFile(dest, 'w') as dest_zip:
         for info in source_zip.infolist(): # Iterate over each file in zip
-            dest_zip.writestr(info, source_zip.read(info), compression)
+            if args.deterministic:
+                deterministic_info = ZipInfo(info.filename)
+                deterministic_info.create_system = 0  # everything else is fixed
+                newinfo = deterministic_info
```

Comment:
Thanks for the contribution.
Can we eliminate the "deterministic_info" variable by using "newinfo" at line 40?